### PR TITLE
Added more slangs

### PR DESCRIPTION
**Pull request by grand-rick001**
Closes #

### What new changes did you make? Tick all applicable boxes
- [x] Added new abbreviation
- [ ] Fixed something in the source code
- [ ] Added a new feature
- [ ] Fixed the docs (README.md, CONTRIBTUING.md, etc)

### CODE REVIEW

**Comment by Njong392** [comment]
I'm sorry, but you didn't make any new changes in this pull request. Is there something you were meaning to add?

**Comment by grand-rick001** [comment]
Yeah, but someone beat me to it, no worries I created another PR for another new abbreviation.

**Comment by grand-rick001** [comment]
This new abbreviation addition should resolve the closed PR.

File: public/server/db/faq.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "definition": "Frequently Asked Questions"
```
Comment (Njong392) [inline public/server/db/faq.json:2]: Unfortunately, we can't add formal definitions like these to the records. Only slang.

File: public/server/db/pvp.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "definition": "Player Versus Player"
```
Comment (Njong392) [inline public/server/db/pvp.json:2]: This one is good though.

File: public/server/db/faq.json (hunk 1 above)
Comment (grand-rick001) [reply public/server/db/faq.json]: Okay, I removed it.

**Comment by Njong392** [comment]
Noice! Thank you :bomb:

**Pull request merged**